src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -583,6 +583,18 @@ internal static byte[] ConvertIntToByteArray(int dwInput)
             return rgbOutput;
         }
 
+        internal static int ConvertByteArrayToInt(byte[] input)
```

Comment:
Did you try using BigInt first or some other class? @terrajobst @KrzysztofCwalina do you perhaps know if we have anything which does something similar in the framework?